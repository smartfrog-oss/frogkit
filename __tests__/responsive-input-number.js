import { mount } from 'vue-test-utils'
import ResponsiveInputNumber from '@/components/responsive-input-number'

describe('InputNumber component', () => {

  it('should render component and match snapshot', () => {
    const wrapper = mount(ResponsiveInputNumber, { propsData: { ariaLabel: 'quantity' } })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
