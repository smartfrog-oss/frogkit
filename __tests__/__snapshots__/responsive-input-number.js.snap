// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputNumber component should render component and match snapshot 1`] = `
<section class="fk-responsive-input">
  <input-number min="1" max="3" size="small" value="1" aria-label="quantity" class="fk-responsive-input__input"></input-number> <select options="[object Object],[object Object],[object Object]" value="1" aria-label="quantity" class="fk-responsive-input__select--small"></select>
</section>
`;
